modules/hybridBidAdapter.js

```diff
@@ -0,0 +1,193 @@
+import * as utils from '../src/utils'
+import { registerBidder } from '../src/adapters/bidderFactory'
+import { auctionManager } from '../src/auctionManager'
+import { BANNER, VIDEO } from '../src/mediaTypes'
+import {Renderer} from '../src/Renderer';
+
+const BIDDER_CODE = 'hybrid';
+const DSP_ENDPOINT = 'https://hbe198.hybrid.ai/prebidhb';
+const TRAFFIC_TYPE_WEB = 1;
+const PLACEMENT_TYPE_BANNER = 1;
+const PLACEMENT_TYPE_VIDEO = 2;
+const TTL = 60;
+const RENDERER_URL = '//acdn.adnxs.com/video/outstream/ANOutstreamVideo.js';
+
+const placemenTypes = {
+  'banner': PLACEMENT_TYPE_BANNER,
+  'video': PLACEMENT_TYPE_VIDEO
+};
+
+function buildBidRequests(validBidRequests) {
+  return utils._map(validBidRequests, function(validBidRequest) {
+    const params = validBidRequest.params;
+    const bidRequest = {
+      bidId: validBidRequest.bidId,
+      transactionId: validBidRequest.transactionId,
+      sizes: validBidRequest.sizes,
+      placement: placemenTypes[params.placement],
+      placeId: params.placeId
+    };
+
+    return bidRequest;
+  })
+}
+
+const outstreamRender = bid => {
+  bid.renderer.push(() => {
+    window.ANOutstreamVideo.renderAd({
+      sizes: [bid.width, bid.height],
+      targetId: bid.adUnitCode,
+      rendererOptions: {
+        showBigPlayButton: false,
+        showProgressBar: 'bar',
+        showVolume: false,
+        allowFullscreen: true,
+        skippable: false,
+        content: bid.vastXml
+      }
+    });
+  });
+}
+
+const createRenderer = (bid) => {
+  const renderer = Renderer.install({
+    targetId: bid.adUnitCode,
+    url: RENDERER_URL,
+    loaded: false
+  });
+
+  try {
+    renderer.setRender(outstreamRender);
+  } catch (err) {
+    utils.logWarn('Prebid Error calling setRender on renderer', err);
+  }
+
+  return renderer;
+}
+
+function buildBid(bidData) {
+  const bid = {
+    requestId: bidData.bidId,
+    cpm: bidData.price,
+    width: bidData.width,
+    height: bidData.height,
+    creativeId: bidData.bidId,
+    currency: bidData.currency,
+    netRevenue: true,
+    ttl: TTL
+  };
+
+  if (bidData.placement === PLACEMENT_TYPE_VIDEO) {
+    bid.vastXml = bidData.content;
+    bid.mediaType = VIDEO;
+
+    let adUnit = auctionManager.getAdUnits().find(u => u.transactionId === bidData.transactionId);
```

Comment:
Prebid still doesn't support Array.find, you may want to use: `import find from 'core-js/library/fn/array/find';

Reply:
Hi, thanks for your comment. We fixed it.